interface/patient_tracker/patient_tracker.php

```diff
@@ -43,16 +43,35 @@
 
 #define variables, future enhancement allow changing the to_date and from_date 
 #to allow picking a date to review
```

Comment:
This page has an auto refresh mechanism, thus will lose setting if that goes off. Need to make the auto-refresh mechanism and the submit play well together.